XenAdmin/Wizards/PatchingWizard/PatchingWizard_AutomatedUpdatesPage.cs

```diff
@@ -177,9 +178,13 @@ public override void PageLoaded(PageLoadedDirection direction)
                     }//patch
                 }
 
+                //add a revert pre-check action for this pool

```

Comment:
I think we should add a final action for a pool only if there are any prechecks to revert for that pool